vispy/visuals/colorbar.py

```diff
@@ -679,5 +679,5 @@ def size(self, size):
         elif height == 0.:
             raise ValueError("height must be non-zero, not %s" % (height, ))
 
-        self._halfdim = (width, height)
+        self._halfdim = (width / 2, height / 2)
```

Comment:
If they're passed in as ints in Python 2 and then you do `/ 2` on them, it'll return the integer value for `/ 2`. What I mean is more like this:

```python
...
if self.orientation in ["top", "bottom"]:
            (width, height) = size
        else:
            (height, width) = size
width = float(width)
height = float(height)
...
```

That way, Python 2 will be happy if you do `/ 2` on them and return the right value. Python 3 just plain won't care.